src/test/ui/fully-qualified-type/fully-qualified-type-name2.stderr

```diff
@@ -7,12 +7,12 @@ LL |     return x;
    |            ^ expected enum `y::Foo`, found enum `x::Foo`
    |
    = note: enum `x::Foo` and enum `y::Foo` have similar names, but are actually distinct types
-note: enum `x::Foo` is defined in the current crate.
+note: enum `x::Foo` is defined in module ::x of the current crate
```

Comment:
In normal rust code, this means "the external crate x"; having it be different from what the diagnostic means seems confusing.
```suggestion
note: enum `x::Foo` is defined in module `crate::x` of the current crate
```

Reply:
good point. I can think of some ways to word it:
- ```note: enum `x::Foo` is defined in module `crate::x` ```
- ```note: enum `x::Foo` is defined in module `whatever_the_crate_name::x` ```
 - ```note: enum `x::Foo` is defined in module `crate::x` of the current crate```
- ```note: enum `x::Foo` is defined in module `whatever_the_crate_name::x` of the current crate```

Personally I'm leaning towards the second, but I am unsure what is clearest to a new user.

Reply:
The second has the same problem though - crates don't know their own names, so `whatever_the_crate_name::x` won't compile if you copy it into the source code.

Reply:
Ohh, that's true. I forgot people might want to copy paste it. I have updated it to print as you suggested above.